src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -929,6 +929,12 @@ configurationRegistry.registerConfiguration({
 			tags: ['notebookLayout'],
 			default: {}
 		},
+		[NotebookSetting.formatOnCellExecution]: {
+			markdownDescription: nls.localize('notebook.formatOnCellExecution', "Format a notebook cell upon execution. A formatter must be available, and the editor must not be shutting down."),
```

Comment:
What does the part about shutting down mean?

Reply:
Was carried over from another setting, forgot to revise that string. Don't think there's anything needed other than, `"Format a notebook cell upon execution. A formatter must be available."`